pkg/theiactl/commands/start.go

```diff
@@ -0,0 +1,378 @@
+// Copyright 2022 Antrea Authors
```

Comment:
maybe we should prefix these module names with "policyreco" as in the future we might have "start" command for many jobs.

Reply:
Make sense to me, will handle this after the naming is finalized.